tasks/config-cluster.yml

```diff
@@ -1,21 +1,36 @@
 ---
 # Configure RabbitMQ for cluster
 
+- name: ensure hostnames are in the host file
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ hostvars[item]['ansible_default_ipv4']['address'] }} {{ hostvars[item]['ansible_hostname'] }}"
```

Comment:
When dynamic IPs are used, for example in Amazon AWS, then this breaks your /etc/hosts file completely as the (new) IPs are added again and again.